jscomp/frontend/ppx_entry.ml

```diff
@@ -29,7 +29,9 @@ let rewrite_signature (ast : Parsetree.signature) : Parsetree.signature =
   Ast_config.iter_on_bs_config_sigi ast;
   let ast =
     match !Js_config.jsx_version with
-    | 3 -> Reactjs_jsx_ppx_v3.rewrite_signature ast
+    | 3 ->
+        Reactjs_jsx_ppx.rewrite_signature ~jsxVersion:3 ~jsxModule:""
```

Comment:
It might be duplicated to #5484

Reply:
I want to merge this in master.
Then https://github.com/rescript-lang/rescript-compiler/pull/5484 rebases on top.
A small step, but gets the basic structure into master.

Reply:
Got your point!